### DISCUSSION TITLE
Feature: Tab panel

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.UI#211] [Ideas]
Currently there's a `<uui-tabs>` component and also a `<uui-tab-group>` although list isn't listed in the readme here: https://github.com/umbraco/Umbraco.UI

However when it comes to accessibility I think it is missing a `<uui-tab-panel>` component as well, which is connected to the specific tab.
For example: https://shoelace.style/components/tab-panel

**Comment by bjarnef** [comment]
@iOvergaard not sure how the tabel panel are implemented with tabs in the new backoffice, but it think `<uui-tab-panel>` inside `<uui-tab-group>` would make it simpler to connect each tab to its panel and control the necessary aria attributes, e.g. `aria-selected`, `aria-controls`, `aria-labelledby`, `aria-expanded` etc.

**Reply by bjarnef** [reply]
I started a PR here: https://github.com/umbraco/Umbraco.UI/pull/600

**Comment by iOvergaard** [comment]
Sounds good, @bjarnef, and I agree. Let's build it.
